src/object/srv_obj_migrate.c

```diff
@@ -1257,22 +1285,26 @@ migrate_fetch_update_bulk(struct migrate_one *mrone, daos_handle_t oh,
 			D_GOTO(out, rc);
 	}
 
-	/* The data, rebuilt from replication, there are two cases,
-	 * 1. If it comes from overwritten, then it should have higher epoch than the above
-	 * data, rebuilt from parity.
-	 * 2. If it just partial update, then its epoch should not higher than the highest replicate
-	 * epoch on the parity shard.
-	 *
-	 * So let's choose minimum epoch + 1 as its update epoch, which is higher than case 1 epoch,
-	 * and also in case 2, there should have multiple different replicate recxs with different
-	 * epoch, so min_epoch + 1 should not be higher than the highest epoch anyway.
+	/* The data, rebuilt from replication, needs to keep the same epoch during rebuild,
+	 * otherwise it may mess up the relationship between parity epoch vs data(paritied epoch)
```

Comment:
is paritied a word? I think it should just be parity epoch

Reply:
ah, sure